src/developer/web-api/route.md

```diff
@@ -44,6 +32,8 @@ If you want DHIS2 to _POST_ the request to the route target, use the _POST_ HTTP
 POST /api/routes/postman/run
 ```
 
+For performance reasons, the maximum transfer time for a route response is 5 minutes. A response body from the upstream server exceeding this limit will lead to the DHIS2 server returning a bad gateway error to the client.
```

Comment:
Let's move this down to the timeout section, it's a bit out of context here and confusing since we also talk about a response timeout below.